.storybook/preview.tsx

```diff
@@ -7,7 +7,7 @@ import * as React from 'react';
 const withChakra = (StoryFn: Function) => (
   <ThemeProvider theme={theme}>
     <CSSReset />
-    <div id="story-wrapper" style={{ minHeight: '100vh' }}>
+    <div id="story-wrapper" style={{ minHeight: '100vh', backgroundColor: '#F3F3F3' }}>
```

Comment:
I think this is not what @adrianoresende suggested. He meant to put different background colors on some layout specific use-cases (eg. header/sidebar/body/footer).
Suggest removing this since we will address this on another issue.
```suggestion
    <div id="story-wrapper" style={{ minHeight: '100vh' }}>
```